templates/python39-mlops/mlops.json

```diff
@@ -0,0 +1,5 @@
+{
+  "model_id": "6126df6a78aa5cb610acea36",
```

Comment:
we can replace these model and deployment ids with placeholders so that users get the hint to fill them in by themselves

Reply:
I know, I just meant placeholders like:
```
"model_id": "YOUR_MODEL_ID",
"deployment_id": "YOUR_DEPLOYMENT_ID",
```